source/usage-examples/changeStream.txt

```diff
@@ -34,15 +34,29 @@ The ``watch()`` method accepts an ``options`` object as the second parameter. Re
 section for more information on the settings you can configure with this object.
 
 The ``watch()`` method returns an instance of a `ChangeStream <{+api+}/classes/ChangeStream.html>`__. You can read events from
-change streams by iterating over them or listening for events. Select the tab that corresponds to the way you want to
-read events from the change stream below.
+change streams by iterating over them or listening for events. 
 
-.. include:: /includes/changestream-paradigm-warning.rst
+Select the tab that corresponds to the way you want to
+read events from the change stream:
 
 .. tabs::
 
    .. tab::
-      :tabid: Iterative
+      :tabid: Idiomatic Iteration
+
+      Starting in version 4.12, the ``ChangeStream`` class is an async
+      iterable. With this change, you can use ``for-await`` loops to
```

Comment:
I: I don't _think_ it's accurate to say that a class is an async iterable. Based on [this page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols#the_async_iterator_and_async_iterable_protocols), a class/object implements the async iterable protocol, and then the _object_ is the async iterable. I know this gets weird in JS, which isn't a true OOP language, but maybe we could try:

```suggestion
      Starting in version 4.12, the ``ChangeStream`` class implements the async
      iterable protocol. With this change, you can use ``for-await`` loops to
```

Or:

```suggestion
      Starting in version 4.12, ``ChangeStream`` objects are async
      iterables. With this change, you can use ``for-await`` loops to
```